### DISCUSSION TITLE
similar image search stuck on 100k~ image

### DISCUSSION THREAD

**Discussion by utopictown** [milvus-io/milvus#18866] [Q&A and General discussion]
search() stuck and never return result, no timeout notice either, anyone could figure out what might be the cause?

data is around 100k~, already stuck on probably around 60k ish
my flow to seed them is 1000 item per insert() then create_index() run

dim = 2048
index file size = 1024

index params = {"index_type": "IVF_SQ8", "metric_type":"L2", "params": {"nlist": 16384}}

search params = {"metric_type": METRIC_TYPE, "params": {"nprobe": 16}}

what should i tweak around?

**Comment by yhmo** [comment]
Which milvus version?

**Reply by utopictown** [reply]
2.0.2

**Reply by yhmo** [reply]
It might be a bug of v2.0.2. 
I just use two clients to test your case on v2.1.1. The client_1 repeatedly calls insert() and create_index(), 1000 entities for each insert action. The client_2 repeatedly calls search(). The two clients run at the same time, they work fine.

We have fixed a lot of critical issues in v2.1.1, please upgrade to v2.1.1. By the way, a new minor version v2.1.2 will be released in two days.

**Reply by utopictown** [reply]
thanks will try
